new/LIT6026NagaraBaqweet.xml

```diff
@@ -0,0 +1,85 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6026NagaraBaqweet" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ነገረ፡ በቍኤት፡ ዘቅዱስ፡ አባ፡ ወግሪስ፡</title>
```

Comment:
Take care we already have
LIT2022Monast
LIT2121OntheE = CPG 2451
and
LIT2318Senten = CPG 2435

(I assume the ID you created is actually a doublet to LIT2318Senten)


(see EAE II 457b-458ab) 

********
"
The treatise "On the Eight Thoughts (or: Spirits,
Greek logismoiv) of Evilness" (CPG II, no. 2451) –
..... BritLib Orient. 754, fols.
178r–80v, and Orient. 757, fols. 140r–142v;

.... also found in the
so-called "Monastic Collection", edited and translated
from mss. BritLib Orient. 764 
....

In the same Monastic
Collection the Sentences to Monks (CPG II, no.
2435) are found ...
....
In ms.
BritLib Orient. 754, fols. 182rf., the Sentences
are further followed by three letters from the
correspondence between Lucius (Lukyos) and E.,
whose second letter is divided into 21 sections;
the identification of Lucius with the addressee
of E.’s Treatise to Eulogius (CPG II, no. 2447) is,
however, doubtful (cf. Hausherr 1940)."

Reply:
This is the second letter, thank you for the information - I have no access to EAe at home